pyproject.toml

```diff
@@ -47,7 +49,7 @@ test = [
 source = "nodejs"
 
 [tool.hatch.metadata.hooks.nodejs]
-fields = ["description", "authors", "urls"]
+fields = ["license", "description", "authors", "urls"]
```

Comment:
With the `license` added to `package.json`, we can now ask `hatch-nodejs-version` to pull it into `pyproject.toml` for us